scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java

```diff
@@ -383,6 +413,14 @@ private static void addShutdownMessageHook() {
         Runtime.getRuntime().addShutdownHook(new Thread(new Runnable() {
             @Override
             public void run() {
+
+                //Shutting down boot microservices (notably the IAM microservice)
+                LOGGER.info("Stopping boot microservices...");
+
+                for (Process process : bootMicroservicesProcesses) {
+                    process.destroyForcibly();
```

Comment:
There is nothing to wait for concerning the IAM process, and I want to be sure that IAM is destroyed, otherwise PA cannot be started again. I will investigate further your suggestion, and I will make a new PR for any changes .